addons/test_mail_full/tests/test_sms_composer.py

```diff
@@ -656,3 +656,20 @@ def test_composer_template_context_action(self):
             test_record_2.customer_id, None,
             content="Hello %s ceci est en français." % test_record_2.display_name
         )
+
+    def test_composer_sending_with_no_number_field(self):
```

Comment:
Don't forget to add a docstring explaining the use case you are covering, which issues you are trying to avoid.